docs/integrations/sources/instagram.md

```diff
@@ -83,6 +89,25 @@ AirbyteRecords are required to conform to the [Airbyte type](https://docs.airbyt
 | `array`          | `array`      |
 | `object`         | `object`     |
 
+## Limitations & Troubleshooting
+
+<details>
+<summary>
+Expand to see details about Instagram connector limitations and troubleshooting.
+</summary>
+
+### Connector limitations
+
+#### Rate limiting
+
+Instagram limits the number of requests that can be made at a time, but the Instagram connector handles rate limiting. See Facebook's [documentation on rate limiting](https://developers.facebook.com/docs/graph-api/overview/rate-limiting/#instagram-graph-api) for more information.
```

Comment:
Instagram limits the number of requests that can be made at a time. See Facebook's [documentation on rate limiting](https://developers.facebook.com/docs/graph-api/overview/rate-limiting/#instagram-graph-api) for more information.